pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -400,9 +419,10 @@ func loadBalancerServiceChanged(current, expected *corev1.Service) (bool, *corev
 	updated := current.DeepCopy()
 	changed := false
 
-	// Preserve everything but the AWS LB health check interval annotation &
-	// GCP Global Access internal Load Balancer annotation.
-	// (see <https://bugzilla.redhat.com/show_bug.cgi?id=1908758>).
+	// Preserve everything except:
+	// AWS LB health check interval annotation (see <https://bugzilla.redhat.com/show_bug.cgi?id=1908758>) ,
+	// GCP Global Access internal Load Balancer annotation (see <https://bugzilla.redhat.com/show_bug.cgi?id=1908758>), &
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1908758 isn't applicable to GCP Global Access.
Maybe instead we could use https://issues.redhat.com/browse/NE-518 ?